### PR TITLE
Tweaked freshness security consideration. Closes #15

### LINKED ISSUE #15: Freshness "out-of-scope" vs "not required"
Since the document discusses it, it is by definition, in-scope.

Perhaps change the paragraph to say "is not required", or "is left up to the discretion of protocol designers and implementors" or something similar.

The whole paragraph about freshness probably needs a re-write for consistency since it currently reads like it was written by two authors (which it was).

### PR DESCRIPTION
**Pull request by ounsworth**

### CODE REVIEW

File: draft-ounsworth-csr-attestation.md [hunk 1]
```diff
@@ -409,16 +411,21 @@ synchronized clocks.
...
 considering that non-automated certificate enrollments are often asyncronous,
 and considering the common practice of re-using the same CSR
 for multiple certificate renewals across the lifetime of a key.
 "Freshness" typically implies both asserting that the data was generated
 at a certain point-in-time, as well as providing non-replayability.
+Also, HSMs are typically designed to not allow downgrade of private key storage
+properties; for example if a given key was asserted at time T to have been
+generated inside the hardware boundary and to be non-exportable,
+then it can be assumed that those properties of that key will continue
+to hold into the future, thus making freshness largely moot.
```
Comment (hannestschofenig) [inline draft-ounsworth-csr-attestation.md:424]:
```suggestion
to hold into the future.
```

Comment (hannestschofenig) [reply draft-ounsworth-csr-attestation.md:424]: I see the use of this specification a bit broader and there freshness may be an important aspect, which is why we wrote the document about conveying a nonce in a certificate management protocol

**Review by hannestschofenig — CHANGES_REQUESTED** [review]
The text is good. Just have a few minor text suggestions.

**Comment by ounsworth** [comment]
Oops, I pushed 96aa8ec to the wrong branch. Oh well, I guess it's here now.

**Review by mcr — APPROVED** [review]

**Pull request merged**